toolchains/install_bootstrap/musl/setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```

Comment:
And also crt{begin,end}S.o creation near the bottom. And also, this script now invokes the install_dependencies main.py (at the top) instead of the other way round.